components/telemetry-operator/README.md

```diff
@@ -2,83 +2,22 @@
-
-   The previous command also installs a LogPipeline CR, which configures the Telemetry Fluent Bit to push logs to the Loki backend.
-
-3. After disabling Fluent Bit from the logging chart, delete the unneeded resources:
-
-   ```bash
-   kubectl delete daemonset -n kyma-system logging-fluent-bit
-   kubectl delete configmap -n kyma-system logging-fluent-bit
-   kubectl delete servicemonitor -n kyma-system logging-fluent-bit
-   ```
-
-### Disable Telemetry Operator integration
-
-1. To disable the Telemetry Fluent Bit integration, simply deploy the default version of the logging component, which doesn't contain the Loki LogPipeline:
-
-   ```bash
-   kyma deploy --component logging 
-   ```
-
-2. After installing the chart, disable the Telemetry component so that it does not collect the logs anymore:
-
-   ```bash
-   kubectl delete validatingwebhookconfigurations validation.webhook.telemetry.kyma-project.io
-   kubectl delete servicemonitor -n kyma-system telemetry-operator-metrics
-   kubectl delete deployment -n kyma-system telemetry-operator
-   kubectl delete daemonset -n kyma-system telemetry-fluent-bit
-   kubectl delete service -n kyma-system telemetry-operator-webhook
-   kubectl delete service -n kyma-system telemetry-operator-metrics
-   kubectl delete service -n kyma-system telemetry-fluent-bit
-   kubectl delete rolebinding -n kyma-system telemetry-operator-leader-election-rolebinding
-   kubectl delete clusterrolebinding telemetry-operator-manager-rolebinding
-   kubectl delete clusterrolebinding telemetry-fluent-bit
-   kubectl delete clusterrole telemetry-operator-manager-role
-   kubectl delete clusterrole logpipeline-viewer-role
-   kubectl delete clusterrole logpipeline-editor-role
-   kubectl delete clusterrole telemetry-fluent-bit
-   kubectl delete configmap -n kyma-system telemetry-fluent-bit
-   kubectl delete secret -n kyma-system telemetry-operator-webhook-cert
-   kubectl delete serviceaccount -n kyma-system telemetry-operator
-   kubectl delete serviceaccount -n kyma-system telemetry-fluent-bit
-   ```
+The operator has been bootstrapped with [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) 3.6.0. Additional APIs can also be [added by Kubebuilder](https://book.kubebuilder.io/cronjob-tutorial/new-api.html).
 
 ## Development
 
 ### Prerequisites
-- Install [kubebuilder 3.2.0](https://github.com/kubernetes-sigs/kubebuilder) which is the base framework for this controller
-- Install [kustomize](https://github.com/kubernetes-sigs/kustomize) which lets you customize raw, template-free `yaml` files during local development
-- Install [Golang 1.18](https://golang.org/dl/) or newer (for local execution)
+- Install [kubebuilder 3.6.0](https://github.com/kubernetes-sigs/kubebuilder) which is the base framework for this controller
+- Install [kustomize](https://github.com/kubernetes-sigs/kustomize) which lets you customize raw, template-free `yaml` files during local development (see `kustomize` make target)
```

Comment:
```suggestion
- Install [kustomize](https://github.com/kubernetes-sigs/kustomize), which lets you customize raw, template-free YAML files during local development (see `kustomize` make target).
```

What does "(see `kustomize` make target)" mean??

Reply:
we have a dedicated make target to install kustomize. So executing `make kustomize` installs kustomize on the fly